src/Core/Grid/Definition/Factory/OrderMessageGridDefinitionFactory.php

```diff
@@ -185,6 +192,12 @@ protected function getBulkActions()
                     ->setOptions([
                         'submit_route' => 'admin_order_messages_bulk_delete',
                         'confirm_message' => $this->trans('Delete selected items?', [], 'Admin.Notifications.Warning'),
+                        'modal_options' => new ModalOptions([
```

Comment:
You should use the `BulkDeleteActionTrait::buildBulkDeleteAction` here, it's easier and more consistent with other definitions